src/assets/scripts/Pinecone/Dialog/index.js

```diff
@@ -61,6 +72,15 @@ class Dialog {
 		if ( this.config.question ) {
 			innerHtml += `<p id="q-${unique + 1}">${this.config.question}</p>`;
 		}
+		if ( this.config.input && this.config.inputLabel ) {
+			innerHtml += '<div class="input-group">';
+			innerHtml += `<label for="${this.config.input}">${this.config.inputLabel}</label>`;
+			innerHtml += `<input id="${this.config.input}" type="text" name="${this.config.input}" />`;
```

Comment:
Couple questions (more design related):
* Should there be an explicit close button on the dialog? My expectation is there should be one.
* If the input field is mandatory, I think the initial focus should be on the input field (or second in the tab order if a close button is added as per comment above).
* If the input field is optional, the initial focus can be on "No, don't save".

Thoughts?

Reply:
>     * Should there be an explicit close button on the dialog? My expectation is there should be one.

I thought we had discussed this at some point, but my thinking was that we shouldn't. I was thinking this for a few reasons:

1. The modal is essentially an enhanced version of a native [`confirm()`](https://codepen.io/greatislander/pen/RwNXvVX) dialog, which doesn't provide a close button; you either confirm or cancel (hitting the <kbd>esc</kbd> key is effectively the same as cancelling the operation).
2. This guide doesn't mention adding a close button other than the cancel button: https://guide.inclusivedesign.ca/tools/AccessibleDevelopmentTools.html
3. In this case, where a user may have entered text, closing the dialog with a neutral close button as opposed to a specific cancel button might lead some users to think that if they invoke the dialog again, their input will be preserved.

>     * If the input field is mandatory, I think the initial focus should be on the input field (or second in the tab order if a close button is added as per comment above).

Agreed! Fixed in c220f9b.
 
>     * If the input field is optional, the initial focus can be on "No, don't save".